src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
```

Comment:
you should implement these modifiers and add them to html
<img width="741" alt="image" src="https://github.com/mate-academy/layout_calendar/assets/102411130/b14fa17e-e30d-4053-adcb-a51f96970535">